docs/Concepts/TLS-Communication.md

```diff
@@ -0,0 +1,15 @@
+description: Orion TLS 
+<!--- END of page meta data -->
+
+# TLS Communication
+
+Orion supports TLS to secure communication between a client
+(for example Hyperledger Besu), and other Orion nodes.
+
+![Orion TLS](../images/Orion_TLS.png)
```

Comment:
This diagram is going to be updated with the @jakehaugen version isn't it?  Maybe create an issue to capture this and we don't forget it.